packages/web/src/questionnaire/trackResults.ts

```diff
@@ -116,20 +116,29 @@ export const results = {
         },
         required: false,
         type: 'textarea',
-        rows: 6,
+        rows: 8,
         // for debugging purposes
-        defaultValue: `Bonjour,\n
-Mon entreprise a une activité de type "{secteur}".
-J'aimerais bénéficier du dispositif "{titreAide}". \n
+        defaultValue: `Bonjour,
+
+Mon entreprise a une activité de "{label_sectors}{secteur}".
```

Comment:
issue: I just tested this branch locally. For now, the result doesn't seem completly finished or completly understable from an user perspective I think. 

<img width="496" alt="Capture d’écran 2023-10-25 à 15 10 48" src="https://github.com/betagouv/transition-ecologique-entreprises-widget/assets/21986727/9f1098ea-721f-4291-9fd1-3d919203325d">

Within `<>` some suggestions to invite the user to fill the blanks : 

- `Mon entreprise a une activité <de type> "{label_sectors}"`
- `[...] car j'ai pour projet de <...>`
- `J'ai besoin d'être accompagné pour <...>` OR `J'ai besoin d'être recontacté pour savoir comment bénéficier de cet <{program_type}>`

Reply:
thought: I would add that in the case the user doesn't enter a SIRET number - i.e. the user makes a choice from `trackSectors.ts` track to select a sector, skipping `trackSiret.ts` - there is no corresponding key nor value for the store's `label_sectors` value  at the end of the questionnaire